### PR TITLE
Do not return content if the module is disabled

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
In case of module disabled / uninstalled, the module mustn't return anything.

### CODE REVIEW

File: contactform.php [hunk 1]
```diff
@@ -57,6 +57,9 @@ public function install()
 
     public function renderWidget($hookName = null, array $configuration = [])
     {
+        if (!$this->active) {
+            return;
```
Comment (eternoendless) [inline contactform.php]: Shouldn't it return an empty string?

Comment (Quetzacoalt91) [reply contactform.php]: It can be, but `null` works as well

**Comment by marionf** [comment]
It's ok but currently the module isn't installed by default.
After this modification, it would be better if it's installed

**Comment by eternoendless** [comment]
Thank you @Quetzacoalt91

**Comment by marionf** [comment]
Fix this issue: http://forge.prestashop.com/browse/BOOM-3170

**Pull request merged**